### PR TITLE
Add WTF60

### PR DESCRIPTION
**Pull request by nooges**
## Description

Add WTF60 keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/wtf60/config.h [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xCB10
+#define PRODUCT_ID      0x1337
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Keebio
+#define PRODUCT         WTF60
+#define DESCRIPTION     Mirrored 60% keyboard
```
Comment (fauxpark) [inline keyboards/keebio/wtf60/config.h:27]:
```suggestion
```

File: keyboards/keebio/wtf60/readme.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# WTF60
+
+A mirrored 60% keyboard to mess with your brain.
+
+Keyboard Maintainer: [nooges/bakingpy](https://github.com/nooges)  
+Hardware Supported: WTF60 PCB w/ATmega32u4  
+Hardware Availability: [Keebio](https://keeb.io)
```
Comment (fauxpark) [inline keyboards/keebio/wtf60/readme.md:7]:
```suggestion
* Keyboard Maintainer: [nooges/bakingpy](https://github.com/nooges)
* Hardware Supported: WTF60 PCB w/ATmega32u4
* Hardware Availability: [Keebio](https://keeb.io)
```

File: keyboards/keebio/wtf60/readme.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    make keebio/wtf60:default:dfu
```
Comment (fauxpark) [inline keyboards/keebio/wtf60/readme.md:11]:
```suggestion
    make keebio/wtf60:default

Flashing example for this keyboard:

    make keebio/wtf60:default:flash
```

File: keyboards/keebio/wtf60/rules.mk [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (fauxpark) [inline keyboards/keebio/wtf60/rules.mk:11]:
```suggestion
```

File: keyboards/keebio/wtf60/rules.mk [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = yes          # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
```
Comment (fauxpark) [inline keyboards/keebio/wtf60/rules.mk:30]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/keebio/wtf60/config.h [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+#define RGB_DI_PIN E6
+#ifdef RGB_DI_PIN
+  #define RGBLED_NUM 16
+  #define RGBLIGHT_HUE_STEP 8
+  #define RGBLIGHT_SAT_STEP 8
+  #define RGBLIGHT_VAL_STEP 8
+  #define RGBLIGHT_LIMIT_VAL 255 /* The maximum brightness level */
+  #define RGBLIGHT_SLEEP  /* If defined, the RGB lighting will be switched off when the host goes to sleep */
+/*== all animations enable ==*/
+  #define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/keebio/wtf60/config.h:59]: nit pick: 
```suggestion
#    define RGBLED_NUM 16
#    define RGBLIGHT_HUE_STEP 8
#    define RGBLIGHT_SAT_STEP 8
#    define RGBLIGHT_VAL_STEP 8
#    define RGBLIGHT_LIMIT_VAL 255 /* The maximum brightness level */
#    define RGBLIGHT_SLEEP  /* If defined, the RGB lighting will be switched off when the host goes to sleep */
/*== all animations enable ==*/
#    define RGBLIGHT_ANIMATIONS
```

File: keyboards/keebio/wtf60/wtf60.h [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+#define LAYOUT_60_ansi( \
+    kA15, kA14, kA13, kA12, kA11, kA10, kA9, kA8, kA7, kA6, kA5, kA4, kA3, kA1, \
+    kB14, kB13, kB12, kB11, kB10, kB9,  kB8, kB7, kB6, kB5, kB4, kB3, kB2, kB1, \
+    kC14, kC13, kC12, kC11, kC10, kC9,  kC8, kC7, kC6, kC5, kC4, kC3, kC1, \
+    kD14, kD12, kD11, kD10, kD9,  kD8,  kD7, kD6, kD5, kD4, kD3, kD1, \
+    kE14, kE13, kE12,             kE7,                      kE4, kE3, kE2, kE1 \
+) \
+LAYOUT_full_wtf( \
```
Comment (drashna) [inline keyboards/keebio/wtf60/wtf60.h:43]: Please no recursive layouts.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/keebio/wtf60/wtf60.h (hunk 7 above)
Comment (nooges) [reply keyboards/keebio/wtf60/wtf60.h]: Done, killed it. 🔪

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**